lambda/translprim.ml

```diff
@@ -901,8 +901,8 @@ let lookup_primitive loc ~poly_mode ~poly_sort pos p =
         Primitive (Patomic_exchange {immediate_or_pointer=Pointer}, 2)
     | "%atomic_compare_exchange" ->
         Primitive (Patomic_compare_exchange {immediate_or_pointer=Pointer}, 3)
-    | "%atomic_cas" ->
```

Comment:
I do not think we should change names of builtins that exist upstream, if possible. Adding new ones seems fine. Renaming existing `Lambda` primitive constractor here is not so bad (although I'd prefer not to), we have already changed it by adding an argument.

Reply:
Sad but alright